scripts/common/docker.sh

```diff
@@ -98,10 +98,17 @@ function uninstall_docker() {
     if [ "$(docker ps -aq | wc -l)" != "0" ] ; then
         docker ps -aq | xargs docker rm -f || true
     fi
+
+    # if docker is not running, skip this process
+    if ! systemctl is-active --quiet docker; then
+        echo "Docker not running, not uninstalling."
+        systemctl disable docker.service
+        return
```

Comment:
why return here and not remove docker and uncordon the node etc...

Reply:
yeah probably a better solution